config/default/layout_builder_styles.style.headline_all_caps.yml

```diff
@@ -0,0 +1,12 @@
+uuid: 62c70a56-e25d-4e66-9e2c-ed4b1cb5559d
+langcode: en
+status: true
+dependencies: {  }
+id: headline_all_caps
+label: 'Headline all caps'
```

Comment:
I'm going to adjust these labels in #2075. This will need to be adjusted after that is merged.